pom.xml

```diff
@@ -47,6 +64,14 @@
             <version>6.0.4</version>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/org.apache.commons/commons-dbcp2 -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
```

Comment:
http://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-sql.html#boot-features-connect-to-production-database

Seems like it provides connection pool by default together with jpa starter dependency. So no need in this dependency.